sig-scalability/annual-report-2021.md

```diff
@@ -4,79 +4,66 @@
+     - Added support for modules in our tests
+     - Started measuring availability of api-server
+     - Added support for measuring cilium propagation delay, dns latency
 
+3. KEP work in 2021:
    - Stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.stable
+     - None
    - Beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
+     - [1040 - Priority and Fairness for API Server Requests](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/1040-priority-and-fairness/README.md) - 1.23
    - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
+     - [647 - APIServer Tracing](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/647-apiserver-tracing/README.md) - 1.22
+     - [1669 - Proxy Terminating Endpoints](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/1669-proxy-terminating-endpoints/README.md) - 1.22
+     - [2464 - Kubetest2 CI Migration](https://github.com/kubernetes/enhancements/blob/master/keps/sig-testing/2464-kubetest2-ci-migration/README.md) - 1.21
    - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+     - None
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - Each subproject could benefit from additional hands. However, the Scalability Test Frameworks and Scalability and Performance tests and validation are the ones where we can grow contributors - the other require both very deep and wide understanding of Kubernetes before making reasonable contributions.
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - We care mostly about metrics for our main repository [perf-tests]. We measure them through [devstats]. We keep track of number of reviewers, opened issues and PRs merged.
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - Yes, [CONTRIBUTING.md] points to maintained list of issues with label `good first issue`.
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - We don't have any special requirements for reviewers/approvers.
```

Comment:
I know you mentioned this above, but it might be worth calling out again down here that core areas of sig-scalability require deep and wide understanding of the project.

For the CONTRIBUTING.md doc If someone did want to start down the track of growing in sig-scalability, what would be a good entry point for them? is there any background knowledge or experience that might be useful? like what would you put in a job posting if looking to hire someone to help.

Reply:
Added it here.

Sounds good, I will add "job posting" like description to CONTRIBUTING.md